pybase/client.py

```diff
@@ -181,6 +185,59 @@ def get(self, table, key, families={}, filters=None):
             e._handle_exception(self, dest_region=dest_region)
             # Everything should be dandy now. Repeat the request!
             return self.get(table, key, families=families, filters=filters)
+    
+    def get_many(self, table, keys, families=None):
+        """
+        get row or specified cell with optional filter for all provided keys
+        :param table: hbase table
+        :param key: list of row key
+        :param families: (optional) specifies columns to get,
+          e.g., {"columnFamily1":["col1","col2"], "colFamily2": "col3"}
+        :return: tuple of (list of responses with cells, list of exceptions that occurred)
+        """
+        if len(keys) == 0:
+            return []
+
+        grouped_by_server = defaultdict(lambda: defaultdict(list))
+        for key in keys:
+            dest_region = self._find_hosting_region(table, key)
+            # we must call each region server, which can server many key ranges
+            grouped_by_server[dest_region.region_client.host][dest_region].append(key)
```

Comment:
The format for using this call is pretty straightforward but context if you're not too familiar with hbase:

Each region server hosts (in our case, many) range of keys for a given table. We need to organize each key into the appropriate region server key range. Once we've done that, we can re-group these by server and send a request for all keys matching any key range supported by that server.

Reply:
brilliant